infra/wca_on_rails/production/variables.tf

```diff
@@ -108,3 +108,9 @@ variable "shared" {
   })
   description = "The shared resources between Environments"
 }
+
+variable "WRC_WEBHOOK_URL" {
+  description = "The URL to send delegate report webhook notifications for WRC to"
+  type = string
+  default = "https://joba.me/wca/reports_webhook"
```

Comment:
Summoning @FinnIckler regarding Terraform specifics but I feel like we have different files for the different envs and we don't generally use the default value here...?

Reply:
yeah, I will create it in the different files